101-aci-linuxcontainer-public-ip/azuredeploy.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {    
+  },
+  "variables": {
+    "name": "acilinuxpublicipcontainergroup",
+    "image": "microsoft/aci-helloworld",   
+    "port": "80",
```

Comment:
That's mostly for unnecessary parameters that folks use such as nic name, lb name. These seem like they would be useful to have for choice when deploying the template and affect the container directly